sme_finance_application_schema/batch_application_v1

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema":"http://json-schema.org/draft-04/schema#",
+    "title":"Finance Application Batch submission wrapper",
+    "description":"A wrapper to allow multiple Finance Applications to be submitted",
+    "additionalProperties": false,
+    "type": "object",
+    "required": [
+        "finance_applications"
+    ],
+    "properties": {
+        "batch_unique_identifier": {
+            "description":"A unique identifier for the batch provided by the organisation referring the applicants",
+            "type":"string"
```

Comment:
@raoulsullivan I'd recommend a regexp for this one, but we can improve that later